xml/System/DateTimeOffset.xml

```diff
@@ -6064,7 +6064,7 @@ Strings that do not specify a UTC offset are assumed to have the offset of the l
         <ReturnType>System.DateTimeOffset</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The value of this constant is equivalent to 00:00:00.0000000 UTC, January 1, 1970, in the Gregorian calendar. <xref:System.DateTimeOffset.UnixEpoch/> defines the point in time when Unix time is equal to 0.</summary>
```

Comment:
The build is going to fail due to this line because `<xref:>` is for markdown text. Xml tags should use `<see cref>`:

```suggestion
        <summary>The value of this constant is equivalent to 00:00:00.0000000 UTC, January 1, 1970, in the Gregorian calendar. <see cref="F:System.DateTimeOffset.UnixEpoch" /> defines the point in time when Unix time is equal to 0.</summary>
```